File: testresources/java/accuracy/src/domain/indirect/violatingfrom/AccessObjectReferenceIndirect_WithinIfStament.java
package domain.indirect.violatingfrom;

import domain.indirect.intermediate.ServiceTwo;

public class AccessObjectReferenceIndirect_WithinIfStament {
	
	private ServiceTwo st = new ServiceTwo();

	public AccessObjectReferenceIndirect_WithinIfStament(){
		if (st.getServiceOne() != null) {
			String s = "wrong";
			System.out.println(s);}
	}
}

File: src/test/resources/csharp/benchmark/Domain/Indirect/ViolatingFrom/AccessObjectReferenceIndirect_WithinIfStament.cs
namespace Domain.Indirect.ViolatingFrom
{
	public class AccessObjectReferenceIndirect_WithinIfStament
	{
        private Domain.Indirect.Intermediate.ServiceTwo st = new Domain.Indirect.Intermediate.ServiceTwo();

		public AccessObjectReferenceIndirect_WithinIfStament()
		{
			if (st.GetServiceOne() != null)
			{
				string s = "wrong";
			}
		}
	}
}
